src/e84_geoai_common/debugging.py

```diff
@@ -1,37 +1,49 @@
-import folium  # type: ignore
+import folium  # type: ignore[reportMissingImports]
 from shapely import GeometryCollection
 from shapely.geometry.base import BaseGeometry
 
 from e84_geoai_common.util import timed_function
 
-SEARCH_AREA_STYLE = {"fillColor": "transparent", "color": "#FF0000", "weight": 3}
+SEARCH_AREA_STYLE = {
+    "fillColor": "transparent",
+    "color": "#FF0000",
+    "weight": 3,
+}
 
 
 @timed_function
-def display_geometry(
+def display_geometry(  # type: ignore[reportUnknownPrameterType]
```

Comment:
I don't think we should need this type ignore. All of those types should be available right? Also minor misspelling.

Reply:
If it's related to `g.add_to(m)` calls below I'd prefer to keep them on those specific lines.

Reply:
Good catch. It looks like this ignore isn't needed. I've removed it now.